debian/control.d/control.in

```diff
@@ -18,5 +18,5 @@ Build-Depends: debhelper (>= 6),
     python-protobuf (>= 2.4.1), libprotoc-dev (>= 2.4.1),
     python-simplejson, libboost-thread-dev, libcgroup-dev,
     yapps2-runtime | python-yapps, yapps2,
-    python-pyftpdlib, @BUILD_DEPS@
+    python-pyftpdlib, libck-dev, @BUILD_DEPS@
```

Comment:
There's no `libck-dev` package in Buster, so this needs to be moved into `debian/configure` and added through the `BUILD_DEPS` variable.